neard/src/cli.rs

```diff
@@ -15,7 +15,7 @@ use tracing::metadata::LevelFilter;
 use tracing_subscriber::EnvFilter;
 
 /// NEAR Protocol Node
-#[derive(Clap)]
+#[derive(clap_derive::Parser)]
```

Comment:
I haven't seen the docs for the most recent version of `clap`, but as a reader I'm somewhat confused.
Why do we need `use clap::Parser` if here we derie `clap_derive::Parser`?

Checked the docs, the first example derives `clap::Parser`:
https://docs.rs/crate/clap/3.0.0-rc.7

Reply:
@nikurt Good question.
Note that `use clap::Parser` and `clap_derive::Parser`, share the same name, but one is trait, another is `proc-macro`.

`use clap::Parser` is a trait, which adds `parse` method to ` CliArgs::parse();`

`clap_derive::Parser` is needed for derive.
In Rust all derives / `proc_macro` crates, need to be separate.